aws/resource_aws_waf_rule_test.go

```diff
@@ -34,6 +34,11 @@ func TestAccAWSWafRule_basic(t *testing.T) {
 						"aws_waf_rule.wafrule", "metric_name", wafRuleName),
 				),
 			},
+			{
```

Comment:
Could you add this `TestStep` to all the acceptance tests? That helps us verify that all resource configurations are imported correctly. Thanks!

Reply:
Actually since most of the tests aren't verifying additional configurations other than `TestAccAWSWafRule_geoMatchSetPredicate`, we can probably just get this in as-is.